### ISSUE TITLE
Reactivate Twitter and setup Mastodon

### ISSUE THREAD

**Issue by marcoroth** [ruby-conferences/ruby-conferences.github.io#607]
I'm wondering if it would make sense to reactivate the https://twitter.com/rubyconferences and potentially setup an account on ruby.social to promote and make people more aware of the conferences which are happening. 

Be it when a new one gets announced, a CFP openes, a CFP closes or when the ticket sales start.

I think it would be awesome for visibility to have an "independent" account that can promote and amplify Ruby conferences.

**Comment by adriangohjw** [comment]
i think for a start, we can automate the posting of these new events and meetups via the [twitter API](https://developer.x.com/en/docs/x-api) - it's free up to 1500 posts per month which should be more than sufficient. WDYT?

**Comment by marcoroth** [comment]
I think so too. I'm not sure if @jonallured still has the credentials for it

**Comment by jonallured** [comment]
yes @marcoroth I do have the credentials - email me and we can work something out!

**Comment by adriangohjw** [comment]
Nice, lemme figure out some basic POC on my own first

**Comment by marcoroth** [comment]
Also got an account on ruby.social now: https://ruby.social/@RubyConferences
